decidim-core/app/forms/password_validator.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
```

Comment:
This should't live in the `forms` directory.

There's a separate `validators` directory, see e.g.
https://github.com/decidim/decidim/blob/develop/decidim-core/app/validators/etiquette_validator.rb#L1